src/enterprise-edge/azext_enterprise_edge/_help.py

```diff
@@ -14,7 +14,7 @@
 
 helps['staticwebapp enterprise-edge enable'] = """
     type: command
-    short-summary: Enable the Azure Front Door CDN for a static webapp. For optimal experience and availability please check our documentation https://aka.ms/swaedge
+    short-summary: Enable the Azure Front Door CDN for a static webapp. Enabling enterprise-grade edge requires reregistration for the Azure Front Door Microsoft.CDN resource provider. For more details, please review the documentation available at https://go.microsoft.com/fwlink/?linkid=2184995 . For optimal experience and availability please check our documentation https://aka.ms/swaedge
 """
```

Comment:
Nit:re-registration?
Too Many links in the short-summary move the fwd link about registration to the no-register parameter part of help.